pom.xml

```diff
@@ -22,7 +22,7 @@
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
-            <version>4.12</version>
+            <version>4.13.2</version>
```

Comment:
why do we need to change this version?

Reply:
in previous homework it caused problems. command mvn clean package gave me errors. so in pom file i changed these two settings and it worked. in this home work i made it from the start